server/service.go

```diff
@@ -544,6 +566,8 @@ func (m *BoostService) handleGetPayload(w http.ResponseWriter, req *http.Request
 	// Wait for all requests to complete...
 	wg.Wait()
 
+	// send bid and signed block to relay monitor
+	m.sendAuctionTranscriptToRelayMonitors(AuctionTranscript{Bid: originalBid.response.Data, Acceptance: payload})
```

Comment:
if we change the type signature above:

```suggestion
	m.sendAuctionTranscriptToRelayMonitors(&AuctionTranscript{Bid: originalBid.response.Data, Acceptance: payload})
```